aioredis/commands/set.py

```diff
@@ -43,9 +43,11 @@ def smove(self, sourcekey, destkey, member):
         """Move a member from one set to another."""
         return self.execute(b'SMOVE', sourcekey, destkey, member)
 
-    def spop(self, key, *, encoding=_NOTSET):
-        """Remove and return a random member from a set."""
-        return self.execute(b'SPOP', key, encoding=encoding)
+    def spop(self, key, count=None, *, encoding=_NOTSET):
+        """Remove and return one or multiple random members from a set."""
+        args = [key]
+        count is not None and args.append(count)
```

Comment:
This is a nice Python feature but it also a really smelly code style.
What was the reason not to write it in two lines?
Also code coverage for this line is always 100% regardless of the `const` value (branch coverage doesn't work here as well)